### ISSUE TITLE
found 1 moderate severity vulnerability 

### ISSUE THREAD

**Issue by terehov** [sdawood/json-tots#1]
Awesome library, I was looking for something like that for a whiel and was just about to start writing my own one :-) 

When installing I get the following error from the `jsonpath` dependency you are using: 
```
│ Moderate      │ Sandbox Breakout / Arbitrary Code Execution                  │
├───────────────┼─────────────────────────────
│ Package       │ static-eval                                                  │
├───────────────┼─────────────────────────────
│ Patched in    │ >=2.0.0                                                      │
├───────────────┼─────────────────────────────
│ Dependency of │ json-tots                                                    │
├───────────────┼─────────────────────────────
│ Path          │ json-tots > jsonpath > static-eval                           │
├───────────────┼─────────────────────────────
│ More info     │ https://nodesecurity.io/advisories/548                       │

```

**Comment by sdawood** [comment]
Thanks for the encouragement

Now I've been seeing those everywhere lately, my guess is that npm has recently been calling 'npm audit' when it does 'npm install'

A call to 'npm audit' fix' bumps your packages forward as much as your semVer pattern allows it, and you would get rid of most of the errors, since the fixes are mostly 'patch' version bumps.

Please open that issue in the jspon path GitHub repo, and I'm sure David would respond in a timely fashion, I'll bump my dependencies accordingly.

**Comment by sdawood** [comment]
Also I'm currently happy to also accept issues of the "Feature Request" falvor, feel free to add some of those using [Feature Request] in the subject.

**Comment by sdawood** [comment]
Closes #1
